packages/acceptance-tests/pkg-tests-specs/sources/protocols/git.test.js

```diff
@@ -0,0 +1,22 @@
+// TODO: Fix tests
```

Comment:
Needed to comment it out due to the failure:

![image](https://user-images.githubusercontent.com/10795657/62520273-88d1ca00-b82d-11e9-9acb-666aec531228.png)

@arcanis does that ring a bell? 🤔

Reply:
It turned out that we needed to add a package without *any* dependencies (not even a `devDependencies`). @arcanis, thanks for finding this one: https://github.com/TooTallNate/util-deprecate 💪